_posts/dev/2015-04-05-api.md

```diff
@@ -79,6 +80,11 @@ Updated in 2.5:
 
 - **create** - **Added:** `meetingCameraCap`, `groups`, `disabledFeatures`, `meetingExpireIfNoUserJoinedInMinutes`, `meetingExpireWhenLastUserLeftInMinutes`; **Deprecated:** `learningDashboardEnabled`, `breakoutRoomsEnabled`, `virtualBackgroundsDisabled`.
 
+- **insertDocument** 
```

Comment:
`- **insertDocument** endopoint was first introduced` should suffice here. It's just a list of api changes per BBB release for quick reference